Digizuite/Metadata/ResponseModels/MetaFields/SlaveItemReferenceMetaFieldResponse.cs

```diff
@@ -7,6 +7,7 @@ namespace Digizuite.Metadata.ResponseModels.MetaFields
     {
         public override MetaFieldDataType Type => MetaFieldDataType.SlaveItemReference;
         public ItemType ItemType { get; set; }
-        public int RelatedMetaFieldLabelId { get; set; }
+        public int? RelatedMetaFieldLabelId { get; set; }
+        public int? RelatedMetaFieldId { get; set; }
```

Comment:
I was too fast there. Slave should always have a related field, since it can really only exists as part of a master field